src/consumers/plain_event_consumer.cpp

```diff
@@ -0,0 +1,70 @@
+#include "consumers/plain_event_consumer.hpp"
+
+#include <iomanip>
+#include <iostream>
+
+using namespace events;
+
+uint64_t nanoseconds(time_point timestamp) {
```

Comment:
This function should probably be renamed to something like `get_nanoseconds_from_timestamp`